sql/dql/intro-joins/primary-foreign-keys.md

```diff
@@ -0,0 +1,80 @@
+---
+author: meridian
+
+type: normal
+
+category: must-know
+
+links: 
+  - [Difference between Primary and Foreign Keys](https://www.dotnettricks.com/learn/sqlserver/difference-between-primary-key-and-foreign-key){article}
```

Comment:
there are two options when adding a link

- old (everything should be enclosed in quotes)
```suggestion
  - '[Difference between Primary and Foreign Keys](https://www.dotnettricks.com/learn/sqlserver/difference-between-primary-key-and-foreign-key){article}'
```

- new (yaml syntax)

```suggestion
  - >-
    [Difference between Primary and Foreign Keys](https://www.dotnettricks.com/learn/sqlserver/difference-between-primary-key-and-foreign-key){article}
```

I suggest we go for the new one

Reply:
Will stick to the new format from now on